csrc/executor_utils.cpp

```diff
@@ -479,16 +555,23 @@ void validateAlignedVectorizedFusionInputOutput(
         tv->axis(i)->toString(),
         ", stride: ",
         stride,
-        ", cur_contig_stride ",
-        cur_contig_stride);
+        ", cur_contig_stride: ",
+        cur_contig_stride,
+        ", non contig due to slice: ",
+        non_contig_due_to_slice);
     // If the domain is size-1, the next domain is still considered
     // rightmost.
     still_rightmost =
         still_rightmost && (size == 1 || is_expanded_broadcasting);
     // We do not update cur_contig_stride for size==1 dimensions,
-    // since we have specialized vectorization stride check for them
+    // since we have specialized vectorization stride check for
+    // them. Same for non_contig_due_to_slice.
     if (size != 1) {
```

Comment:
Should this be the following?
```suggestion
    if (size != 1 && !is_expanded_broadcasting) {
```

Reply:
That seems right. Does that mean we could have errors with expanded broadcast? It seems like with an expanded broadcast domain `cur_contig_stride` becomes zero, so the next domain would fail to pass `is_contiguous`.

Reply:
That's true. Probably just because expanded broadcast in inputs are not common, so we have not hit that error?